### ISSUE TITLE
run-windows -release with javascript bundle ?

### ISSUE THREAD

**Issue by adisoftbn** [ReactWindows/react-native-windows#862]
Hello :)

I'm trying to figure how i can include the js bundle when I build the project using cli.

I prefer running it from cli because it generates also an "appx" for testing.
When i build with visual studio, the appx that i build gets very big(from 6mb using cli to 25mb using VS) and I cannot sign it with debug certificate using SignTool.

Best regards,
Adrian

**Comment by rozele** [comment]
Unfortunately we do not yet have a react-native-cli command for this yet.  For now, the process is outlined here: https://github.com/ReactWindows/react-native-windows/blob/master/docs/RunningOnDeviceWindows.md#using-offline-bundle

Basically:
* Generate the bundle with the `react-native bundle` command
* Open your .sln in Visual Studio 2015 and compile and run with DebugBundle or ReleaseBundle configuration.

**Comment by adisoftbn** [comment]
Hello :)

I already used Visual Studio IDE to build react native solution, but there are some huge differences between Visual Studio IDE Release output and **react-native run-windows -release** output. **run-windows**  generates also an appx, output is a lot smaller in size, generates also a **package.map.txt** and also signs the appx, so I can distribute it to friends without having to use Store yet.

I already use the bundle generator. The only issue I have is the possibility to include the pre generated JS Bundle into **run-windows --release** output.

I searched in all the source code and i haven't found a possibility to include that file into the **package.map.txt**, before it generates the appx.

It seems that i cannot  sign a package created by "makeappx" by me with "signtool".

I get: 

```
Done Adding Additional Store
SignTool Error: An unexpected internal error has occurred.
Error information: "Error: SignerSign() failed." (-2147024885/0x8007000b)
```
